modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -645,9 +646,44 @@ private boolean modelIsMutatable(CodegenModel model, Set<String> processed) {
         return isMutatable;
     }
 
-    protected void removePropertiesDeclaredInComposedTypes(Map<String, ModelsMap> objs, CodegenModel model, List<CodegenProperty> composedProperties) {
+    protected final Stream<CodegenModel> getSubModels(Map<String, ModelsMap> models, CodegenModel model)
+    {
+        Stream<CodegenProperty> anyOf = Stream.empty();
+        Stream<CodegenProperty> oneOf = Stream.empty();
+        if(model.getComposedSchemas() != null && model.getComposedSchemas().getAnyOf() != null) {
+            anyOf = model.getComposedSchemas().getAnyOf().stream();
+        }
+        if(model.getComposedSchemas() != null && model.getComposedSchemas().getOneOf() != null) {
```

Comment:
Consider using a ternary here. It's personal preference, but I think it is easier to read as you know the variable is set without having to read the body of an if statement (one which may change over time).

```java
Stream<CodegenProperty> anyOf = model.getComposedSchemas() != null && model.getComposedSchemas().getAnyOf() != null
        ? model.getComposedSchemas().getAnyOf().stream()
        : Stream.empty();
```

Reply:
Fixed as you proposed. Thanks for the feedback